### PR TITLE
Bug/groomer services

### PR DESCRIPTION
**Pull request by Broast42**
## Description ##

Changed the varchar limit to the hours table in groomer profile to allow for us to store the hours object as a string. Refactored groomer services modal and router. Post, put, and delete routes now take a needed query param so we can add edit and delete by the two needed keys. Also used a join table to grab the name of the service along with the id and the price when grabbing all groomer-services by a groomers id 

## Type ##

- [x] This is a bugfix.
- [ ] This is a new feature.
- [ ] This is part of a feature.

## Organization ##

- [ ] Have you linked this pull request to a Trello card?
- [x] Have you named the branch in the format of feature/describe_feature_or_change OR bug/describe_bug_this_is_fixing 

## Review ##

- [ ] Is this a work in progress?
- [x] Is this ready to be reviewed?
- [x] Have you added two reviewers to this pull request?

### CODE REVIEW

File: api/groomerservices/GroomerServicesModel.js [hunk 1]
```diff
@@ -6,7 +6,18 @@ const getAll = async () => {
...
+    .select(
+      'services.service_name',
+      'services.id',
+      'groomer_services.services_price'
+    );
+};
+
+const checkIfExists = async (service_id, groomer_id) => {
+  return await db('groomer_services')
+    .where({ groomer_id: groomer_id, services_id: service_id })
```
Comment (ShawnBatson) [inline api/groomerservices/GroomerServicesModel.js:20]: Nice expansion. It makes more sense now on what it is returning.

File: data/migrations/20201014204709_create_bare_bones_tables.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ exports.up = async (knex) => {
     table.varchar('zip_code').notNull();
     table.varchar('country').notNull();
     table.varchar('about', 5000);
-    table.varchar('hours');
+    table.varchar('hours', 5000);
```
Comment (ShawnBatson) [inline data/migrations/20201014204709_create_bare_bones_tables.js:15]: Sorry about that. Wasn't sure of the format you wanted in hours.

**Review by ShawnBatson — APPROVED** [review]

File: data/migrations/20201014204709_create_bare_bones_tables.js (hunk 2 above)
Comment (Broast42) [reply data/migrations/20201014204709_create_bare_bones_tables.js:15]: No biggie I didn't know I needed more than the default 225 until I tried to insert the string.

**Review by jessicagertig — APPROVED** [review]
Great job formatting the hours, groomer_services table, and services.

**Pull request merged**